installer/tue-install-impl.bash

```diff
@@ -444,13 +449,17 @@ function tue-install-git
         rm "$version_cache_file" 2>/dev/null
     fi
 
-    tue-install-debug "Desired branch: $BRANCH"
-    if [ -n "$BRANCH" ] # Cannot be combined with version-if because this one might not exist
-    then
+    local _try_branch_error_code
+    tue-install-debug "Desired branch(es): ${BRANCH}"
+    for branch in ${BRANCH}
+    do
+        tue-install-debug "Parsed branch '${branch}'"
         _try_branch_res=""
-        _try_branch_git "$targetdir" "$BRANCH"
-        [ -n "$_try_branch_res" ] && res="${res:+${res}\n}$_try_branch_res"
-    fi
+        _try_branch_git "${targetdir}" "${branch}"
+        _try_branch_error_code=$?
+        [ -n "${_try_branch_res}" ] && res="${res:+${res}\n}${_try_branch_res}"
+        [ "${_try_branch_error_code}" -eq 0 ] && break
+    done
```

Comment:
I think we should add a note/caution somewhere in the README/code that this behavior has changed. Consider this: We have the branch list to be `foo bar baz`. If one the repos has both `foo` and `bar` as available branches in it, then the repo is checked out with `foo` and not `bar`. So having multiple branches specified might result in unintended behavior if not used correctly.

Reply:
No because of the reverse order inside the script, it will be checked out to `bar`. But it is good to mention the priority order of the arguments in the docs.